src/components/Search.vue

```diff
@@ -0,0 +1,192 @@
+          v-icon {{ $vuetify.icons.alert }}
+        v-list-tile-content
+          v-list-tile-title No results found for query '{{ searchText }}'
+          v-list-tile-sub-title Modify search parameters or refine your query.
+      v-list-tile(v-show="!loading && showMore && searchResults.length > maxQuickResults",
+          @click="$emit('moreresults', searchParams)")
+        v-list-tile-action
+          v-icon {{ $vuetify.icons.more }}
+        v-list-tile-content
+          v-list-tile-title More
+      //- Skeleton search results shown as "loading" animation
+      v-list-tile(
+          v-show="loading",
+          v-for="i in (maxQuickResults + (showMore ? 1 : 0))",
+          :key="`skeleton-${i}`")
+        v-list-tile-action
+          v-icon.grey--text.text--lighten-1 {{ $vuetify.icons.circle }}
+        v-list-tile-content
+          v-list-tile-title.skeleton.skeleton--text.mb-2(
+              :style="{ width: (60 + (4 * (i % 3))) + '%', height: '10px' }")
+          v-list-tile-sub-title.skeleton.skeleton--text(
+              :style="{ width: (45 - (4 * (i % 2))) + '%', height: '6px' }")
+  v-menu(
+      offset-y,
+      left,
+      content-class="girder-search-arrow-menu",
+      :close-on-content-click="false",
+      v-model="searchOptionsMenu")
+    v-btn(icon, slot="activator")
+      v-icon.mdi-24px {{ $vuetify.icons.settings }}
+    v-card
+      v-card-actions
+        v-layout(column)
+          v-radio-group.my-2(hide-details, v-model="searchMode")
+            v-radio.mb-1(key="text", label="Text Search", value="text")
+            v-radio(key="prefix", label="Prefix Search", value="prefix")
+          v-divider
+          v-checkbox.mt-2(hide-details, v-model="searchTypes", label="User", value="user")
+          v-checkbox.mt-1(hide-details, v-model="searchTypes", label="Folder", value="folder")
+          v-checkbox.mt-1.mb-1(hide-details, v-model="searchTypes", label="Item", value="item")
+</template>
+
+<style lang="scss">
+.girder-searchbar {
+  .v-text-field .v-input__control {
+    min-height: 40px;
+  }
+
+  .v-menu__activator * {
+    cursor: text !important;
+  }
+}
+
+.girder-searchbar-menu {
+  .v-list__tile {
+    height: 40px;
+
+    .v-list__tile__action {
+      min-width: 40px;
+    }
+  }
+
+  .skeleton.skeleton--text {
+    background: linear-gradient(270deg, #e0e0e0, #c7c7c7, #e0e0e0);
+    background-size: 600% 600%;
+    animation: SkeletonShimmer 2s ease infinite;
+
+    @keyframes SkeletonShimmer {
+      0% { background-position: 0% 51%; }
+      50% { background-position: 100% 50%; }
+      100% { background-position: 0% 51%; }
+    }
+  }
+}
+
+// TODO: factor out into common stylesheet if we are to use this elsewhere.
+.girder-search-arrow-menu {
+  transform: translateY(10px);
+  // Override to make the arrow visible
+  overflow: visible;
+  contain: inherit;
+
+  // Remove any shadow that is made visible by above style
+  :first-child {
+    box-shadow: none;
```

Comment:
When I remove this rule I can't tell a difference, maybe it's just really subtle or I'm not on the right browser. What does this actually do?

Reply:
I added this originally. Vuetify v-menu has a shadow automatically and it prevent any kind of overflow to prevent a double shadow situation. The style to make the arrow visible to allow overflow causes the shadow of the v-card to show (technically really subtle to see).